index.html

```diff
@@ -363,7 +363,7 @@ <h4>Computation steps</h4>
                       <ol>
                         <li id="comp_labelledby_set_current" name="step2B.ii.a">Set the <code>current node</code> to the node referenced by the IDREF.</li>
                         <li id="comp_labelledby_recursion" name="step2B.ii.b"><em>LabelledBy Recursion:</em> Compute the text alternative of the <code>current node</code> beginning with the overall <a href="#comp_computation">Computation</a> step. Set the <code>result</code> to that text alternative.</li>
-                        <li id="comp_labelledby_append" name="step2B.ii.c">Append the <code>result</code>, with a space, to the <code>accumulated text</code>.</li>
+                        <li id="comp_labelledby_append" name="step2B.ii.c">Append a space character and the <code>result</code> to the <code>accumulated text</code>.</li>
```

Comment:
Justification for including this change: while adding `id="comp_append"` below, I noticed these "append" steps were actually phrased as "joins" ("append the result with a space" is `+= "result "` not `+= " result"`) so I rephrased for accuracy. If the editors prefer, I can include this separately.

Reply:
I think it's fine to leave in here, it's essentially equivalent to "fixing a typo" IMO